libraries/provision/ansible/playbooks/stop-testserver-windows.yml

```diff
@@ -0,0 +1,28 @@
+- name: Windows - Stop TestServer
```

Comment:
This yml and the fetch-windows-cbl-logs.yml look mostly the same. May be this one should be purely about stopping and the other one purely about fetching the logs

Reply:
done